.travis.yml

```diff
@@ -8,21 +8,28 @@ php:
     - hhvm
 
 matrix:
+    fast_finish: true
     include:
         - php: 5.3
-          env: deps=low
+          env: COMPOSER_FLAGS="--prefer-lowest"
+        - php: 5.6
+          env: SYMFONY_VERSION=2.3.*
+        - php: 5.6
+          env: SYMFONY_VERSION=2.7.*
+        - php: 5.6
+          env: SYMFONY_VERSION=2.8.*@dev
+        - php: 5.6
+          env: SYMFONY_VERSION="3.0.x-dev as 2.8"
     allow_failures:
         - php: hhvm
+        - env: SYMFONY_VERSION=2.8.*@dev
+        - env: SYMFONY_VERSION="3.0.x-dev as 2.8"
 
-env:
-    global:
-        - deps=no
-
-before_install:
-    - composer self-update
+before_script:
+    - composer selfupdate
 
 install:
-    - if [ "$deps" = "no" ]; then composer --prefer-source install; fi;
-    - if [ "$deps" = "low" ]; then composer --prefer-source --prefer-lowest --prefer-stable update; fi;
+    - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
+    - composer update --prefer-source --no-interaction $COMPOSER_FLAGS
```

Comment:
``--no-interaction`` is not needed. Travis already forces composer to be non-interactive: https://github.com/travis-ci/travis-cookbooks/blob/59a4d8b8962209bf0041703ed7348c00cd01ef14/ci_environment/travis_build_environment/files/default/travis_environment_php.sh